### PR TITLE
chore(infra): Add helm-config docker image pushing for ids

### PR DESCRIPTION
**Pull request by peturgq**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -571,8 +571,20 @@ jobs:
           aws-access-key-id: ${{ secrets.ECR_AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.ECR_AWS_SECRET_ACCESS_KEY }}
           aws-region: eu-west-1
+      
+      - name: Docker build image IDS
+        working-directory: infra
+        run: |
+          ./scripts/build-docker-container.sh $DOCKER_TAG
+          echo "COMMENT<<EOF" >> $GITHUB_ENV
+          printf "Affected services are: ${{needs.prepare.outputs.IMAGES}}\n" >> $GITHUB_ENV
```
Comment (pshomov) [inline .github/workflows/push.yml:580]: We should probably clarify for which deployment are the affected services, now that we have two comments. Have to refresh my knowledge on what we do with IDS deployments

**Review by pshomov — COMMENTED** [review]
Seems to me that we are now building twice the same container?

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR closed its doors. Sat around for a day after the stale tag, then reckoned it's not worth the fuss.

**Pull request closed without merging**